pkg/prometheus/agent/operator.go

```diff
@@ -1223,3 +1236,12 @@ func ListOptions(name string) metav1.ListOptions {
 		})).String(),
 	}
 }
+
+func MakeSelectorLabels(name string) map[string]string {
```

Comment:
(nit) it can be private
```suggestion
func makeSelectorLabels(name string) map[string]string {
```